### ISSUE TITLE
Cross-entropy metric

### ISSUE THREAD

**Issue by MaxHalford** [creme-ml/creme#68]
We recently added a `CrossEntropy` loss function. It would be nice to also have a `CrossEntropy` in the `metrics` module. This would inherit from `MultiClassificationMetric` and reuse the `CrossEntropy` from the `optim` module, much like `LogLoss` from `metrics` uses `LogLoss` from `optim`.
